ui/apps/platform/cypress/integration/clusters/init-bundles/InitBundles.helpers.js

```diff
@@ -0,0 +1,115 @@
+import { interactAndWaitForResponses } from '../../../helpers/request';
+import { visit } from '../../../helpers/visit';
+
+const pagePath = '/main/clusters/init-bundles';
+const formSearch = '?action=create';
```

Comment:
nit: What do you think of naming this variable `formAction`?

(While it's technically the "search" part of the URL, this name confused me when I first reviewed the code.)